### ISSUE TITLE
Installation succeeds, functions fail

### ISSUE THREAD

**Issue by wholmgren** [NREL/bifacial_radiance#223]
I gave this another shot and I was able to install RADIANCE on an older mac that has XQuartz. Installing `bifacial_radiance` is no problem. But I'm not able to successfully run `demo.genCumSky(demo.epwfile)` (from intro examples) 

```
In [6]: demo.genCumSky(demo.epwfile)
message: /bin/sh: gencumulativesky: command not found
Out[6]: 'skies/cumulative.rad'

In [8]: !echo $PATH
/Users/holmgren/miniconda3/envs/rad/bin:/Users/holmgren/.local/bin:/Users/holmgren/miniconda3/condabin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/git/bin:/usr/texbin

In [14]: !ls ~/.local/bin | grep gen
genblinds
genbox
genclock
gencumulativesky.exe
gendaylit
gendaymtx
genprism
genrev
gensky
gensurf
genworm
```

Installation STEP 2 says to put `gencumulativesky.exe` in `/usr/local/radiance/bin/`. I installed `radiance` in `~/.local`. I tried to put the `exe` in both `~/.local/bin` and `~/.local/lib/ray`. I don't have any directory named `radiance`.

~I also cannot run `bifacial_radiance.gui()`~ (from readme - presumably should also be in intro examples). EDIT: this gui issue was fixed by cloning the github repo and installing from source (version `0.3.3.1+52.g221502f`)

```
In [11]: import bifacial_radiance

In [12]: bifacial_radiance.gui()
---------------------------------------------------------------------------
TclError                                  Traceback (most recent call last)
<ipython-input-12-4d7856f91818> in <module>
----> 1 bifacial_radiance.gui()

~/miniconda3/envs/rad/lib/python3.7/site-packages/bifacial_radiance/gui.py in gui()
   1779
   1780 def gui():
-> 1781     root = Window()
   1782     # bring window into focus
   1783     root.lift()

~/miniconda3/envs/rad/lib/python3.7/site-packages/bifacial_radiance/gui.py in __init__(self)
   1763         ## IMAGE STUFF
   1764         #imagevariables_frame
-> 1765         image_fixed = PhotoImage(file=os.path.join(IMAGE_PATH,'fig1_fixed_small.gif'))
   1766         image_tracked = PhotoImage(file=os.path.join(IMAGE_PATH,'fig2_tracked_small.gif'))
   1767         buttonImage = Button(imagevariables_frame, image=image_fixed)

~/miniconda3/envs/rad/lib/python3.7/tkinter/__init__.py in __init__(self, name, cnf, master, **kw)
   3543         Valid resource names: data, format, file, gamma, height, palette,
   3544         width."""
-> 3545         Image.__init__(self, 'photo', name, cnf, master, **kw)
   3546     def blank(self):
   3547         """Display a transparent image."""

~/miniconda3/envs/rad/lib/python3.7/tkinter/__init__.py in __init__(self, imgtype, name, cnf, master, **kw)
   3499                 v = self._register(v)
   3500             options = options + ('-'+k, v)
-> 3501         self.tk.call(('image', 'create', imgtype, name,) + options)
   3502         self.name = name
   3503     def __str__(self): return self.name

TclError: couldn't open "/Users/holmgren/miniconda3/envs/rad/lib/python3.7/site-packages/bifacial_radiance/images/fig1_fixed_small.gif": no such file or directory
```

I don't have the time to help debug these issues more than this. Sorry.

**Comment by shirubana** [comment]
Sorry this has been so painful, really appreciate your attempts.

I think the [instructions ](
https://bifacial-radiance.readthedocs.io/en/latest/installation.html) say to move the linux executable ~ I can see how this is misleading. I guess it is technically a linux executable but it does not end on .exe, it is just " gencumulativesky "   . I'll update the instructions with a clarification for this too.

**Comment by cdeline** [comment]
Fixed in PR #235
